CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Change Log
 **Upgrade notes:**
+- **Decidim::Verifications::MembersPicker** After installing and activating the verification module, review the [README](https://github.com/gencat/decidim-verifications-members_picker) to execute the necessary actions
```

Comment:
I'd link to the specific section https://github.com/gencat/decidim-verifications-members_picker/blob/master/README.md#usage

Reply:
```suggestion
- **Decidim::Verifications::MembersPicker** After installing and activating the verification module, review the [README usage section](https://github.com/gencat/decidim-verifications-members_picker/blob/0.0.2/README.md#usage) to execute the necessary actions
```